test-interface/src/main/scala/scala/scalanative/testinterface/signalhandling/SignalConfig.scala

```diff
@@ -0,0 +1,59 @@
+package scala.scalanative.testinterface.signalhandling
+
+import scalanative.runtime.Platform._
+import scala.scalanative.libc.stdlib._
+import scala.scalanative.libc.signal._
+import scala.scalanative.unsafe._
+
+private[testinterface] object SignalConfig {
+  def setDefaultHandlers(): Unit = {
+
+    /* Default handler for signals like SIGSEGV etc.
+     * Since it needs to be able to handle segmentation faults, it has to
+     * exit the program on call, otherwise it will keep being called indefinetely.
+     * This and IO operations in handlers leading to unspecified behavior mean we
+     * can only communicate through exitcode values. In bash programs,
+     * exitcode > 128 signifies a fatal signal n, where n = exitcode - 128.
+     * This is the convention used here.
+     */
+    val defaultHandler = CFuncPtr1.fromScalaFunction { (sig: CInt) =>
+      exit(128 + sig)
+    }
+
+    def YELLOW = "\u001b[0;33m"
+    def RESET = "\u001b[0;0m"
+
+    def setHandler(sig: CInt): Unit = {
+      if (signal(sig, defaultHandler) == SIG_ERR)
+        println(
```

Comment:
We can try to print it to standard error
```suggestion
        Console.err.println(
```